cmd/qat_plugin/qat_plugin.go

```diff
@@ -210,12 +211,17 @@ func (dp *devicePlugin) scan() (deviceplugin.DeviceTree, error) {
 	for _, driver := range append(dp.kernelVfDrivers, dp.dpdkDriver) {
 		files, err := ioutil.ReadDir(path.Join(dp.pciDriverDir, driver))
 		if err != nil {
-			return nil, fmt.Errorf("Can't read sysfs for driver %s: %+v", driver, err)
+			fmt.Printf("Driver %s not available: Skipping\n", driver)
+			continue
 		}
 
 		n := 0
 		for _, file := range files {
```

Comment:
This code change was to address the issue described here https://github.com/intel/intel-device-plugins-for-kubernetes/issues/80